NDTensors/src/Unwrap/test/runtests.jl

```diff
@@ -40,8 +41,10 @@ include("../../../test/device_list.jl")
   @test typeof(Ea) == Exposed{m_type,LinearAlgebra.Adjoint{e_type,m_type}}
 
   o = dev(randn(elt, 1))
-  expose(o)[] = 2
-  @test expose(o)[] == 2
+  GPUArraysCore.@allowscalar begin
+    expose(o)[] = 2
+    @test expose(o)[] == 2
+  end
```

Comment:
What is `@allowscalar` needed here? Shouldn't `expose` make that unnecessary?

Reply:
Also we should be able to use `@allowscalar` instead of `GPUArraysCore.@allowscalar` since it is exported.

Reply:
I was missing the `Base.` in `setindex` in `NDTensorsCUDAExt` I added it back in and this code works. Sorry I realized something weird was here but forgot that the `expose` version should work. Thanks!